vue-components/src/components/HelloWorld.vue

```diff
@@ -38,17 +38,23 @@ export default Vue.extend( {
 
 <style lang="scss">
 .wikit-HelloWorld {
+	inset: logical 0 5px 10px;
+	padding-inline: 20px 40px;
+
 	&__headline {
 		margin: 40px 0 0;
```

Comment:
Following https://github.com/wmde/wikit/pull/112/files#diff-e313bcec67163945b0a78b9059513594R32 I assumed there would be some stylelint rule preventing us from using `margin`/`padding`.
I'm a bit surprised that csstools/use-logical does not do that for us, but https://stylelint.io/user-guide/rules/property-blacklist could arguably be used to implement it on our end (not having to would obviously be superior).

Maybe I just misunderstood what the sentence in the ADR means but looking at the compiled CSS there really are no different LTR/RTL versions for those styles.